Apps/Playground/UWP/App.cpp

```diff
@@ -183,8 +183,12 @@ concurrency::task<void> App::RestartRuntimeAsync(Windows::Foundation::Rect bound
     InputManager::Initialize(*m_runtime, *m_inputBuffer);
 
     Babylon::ScriptLoader loader{ *m_runtime, m_runtime->RootUrl() };
+    loader.Eval("document = {}", "");
```

Comment:
Should we consider adding this to `JsRuntime` alongside where it initializes the Window?

Reply:
This is a hack that I hope we can remove later. For some reason, the wrapper for ammo.js and recast.js both expect the `document` object to be there or it will fail to load. Once we have a plugin for the default scripts, this hack can perhaps move with it.